plugins/topology/src/const.ts

```diff
@@ -14,6 +14,8 @@ export const GROUP_PADDING = [
   DEFAULT_GROUP_PAD,
 ];
 
+export const SHOWRESCOUNT = 3;
```

Comment:
@debsmita1 this will be confusing as its not clear for what this max resource count is for. Either we can leave it like this or change it to `MAXSHOWRESCOUNT`.